README.md

```diff
@@ -17,14 +17,14 @@ Example for Maven:
 <dependency>
     <groupId>com.nfl.graphql</groupId>
     <artifactId>mediator</artifactId>
-    <version>1.0.2</version>
+    <version>x.y.z</version>
```

Comment:
Why can't these be driven by environment variables like the README was originally set up?

Reply:
I have checked Git history and as fa as I can tell the version was either 'x.y.z' or '1.0.0'/'1.0.1'/'1.0.2' in this file.
I looked for a solution, but cannot find a way to read the version from the environment or file.
Glitr uses  'x.y.z', too.